cpp/dolfinx/io/ADIOS2Writers.h

```diff
@@ -867,6 +867,330 @@ void vtx_write_mesh_from_space(adios2::IO& io, adios2::Engine& engine,
 }
 } // namespace impl_vtx
 
+/// @privatesection
+namespace impl_checkpoint
+{
+/// Write mesh to file using VTX format
+/// @param[in] io The ADIOS2 io object
+/// @param[in] engine The ADIOS2 engine object
+/// @param[in] mesh The mesh
+template <typename T>
+void write_mesh(adios2::IO& io, adios2::Engine& engine,
```

Comment:
Use more descriptive name if this writes VTX.

Reply:
It does not write VTX. It writes a pure binary file (that cannot be used for visualization, similar to HDF5File in legacy dolfin).

Reply:
> It does not write VTX. It writes a pure binary file (that cannot be used for visualization, similar to HDF5File in legacy dolfin).

So the docstring needs updating? Currently says `/// Write mesh to file using VTX format`.